src/foam/box/HTTPBox.js

```diff
@@ -26,8 +26,9 @@ foam.CLASS({
   ],
 
   imports: [
+    'generator? as ctxGenerator',
```

Comment:
Not sure this was clear from my comment above. The knob is not "correct" or "fast". the `JSONParserStringifier` is correct (and fast). It's just using `parse(JSON.parse(str))` / `JSON.stringify(objectify(o))` rather than `parse(parseString(str))` / `stringify(objectify(o))` [1].

[1] `foam.json`'s parse/stringify is "effectively" `parse(parseString(str))` / `stringify(objectify(o))`, but is not implemented with separation between the two layers in both directions.